src/QMCApp/QMCMain.cpp

```diff
@@ -84,8 +84,19 @@ QMCMain::QMCMain(Communicate* c)
       << "\n  MPI Nodes            = " << OHMMS::Controller->size()
       << "\n  MPI Nodes per group  = " << myComm->size()
       << "\n  MPI Group ID         = " << myComm->getGroupID()
-      << "\n  OMP_NUM_THREADS      = " << omp_get_max_threads()
       << std::endl;
+  #pragma omp parallel
```

Comment:
This output might be better enclosed in a test for nested parallelism (omp_get_nested), and explicitly say something like 'Nested parallelism enabled' (or disabled).  And then only print the number of threads in each level if nesting is enabled.

Reply:
From the standpoint of settings, I would still consider nested parallelism active even if there were only 1 thread in the inner region.  My concern is being able to discern the two cases from the output - if the user is wondering why there is only 1 thread in the inner region - is it because OMP_NUM_THREADS was set that way, or because OMP_NESTED was set that way.
I didn't know about the nested functions and variables being deprecated - that does complicate the decision.  Do you know what is supposed to replace it ?

Reply:
I found the openmp forum thread where you asked about this topic
http://forum.openmp.org/forum/viewtopic.php?f=23&t=2028

One option would be to test the number of threads at the inner level and if it is 1, print that nesting is not enabled. 
Alternately, using omp_get_nested could be used for now, and changed once it's clear where the next version of openmp is heading.

Reply:
In 5.0, you will still need OMP_NESTED to control the nested threading. It is deprecated but not removed.
Nothing to replace it because OMP_NUM_THREADS=list OMP_MAX_ACTIVE_LEVELS should be sufficient.
It is not clear to me how to control exactly in the future. From the code, using two parallel region to feel the environment is solid.

Reply:
Now print "OMP nested threading disabled or only 1 thread on the 2nd level" if the second level size is one.